Assets/Scripts/EnemyAI.cs

```diff
@@ -135,6 +147,7 @@ private void ApproachDestination(float speedMultiplier = 1f)
         LookTowardsDestination();
         if (isGrappled) return;
         if (!thrusterParticleManager.ExhaustTrailActive) thrusterParticleManager.StartExhaustTrail();
+        // TODO: use a rigidbody force instead of changing transform.position?
         transform.position = Vector3.MoveTowards(transform.position, currentDestination, speed * speedMultiplier * Time.deltaTime);
```

Comment:
Use RigidBody.MoveTo (or it's named something similar). there's an equivalent for rotation as well

Reply:
From the documentation: 

> If the rigidbody has isKinematic set to false, it works like transform.position=newPosition and teleports the object to the new position (rather than performing a smooth transition).

https://docs.unity3d.com/ScriptReference/Rigidbody.MovePosition.html